### PR TITLE
Add sig-net to relevant jobs

### PR DESCRIPTION
**Pull request by cmluciano**
Signed-off-by: Christopher M. Luciano <cmluciano@us.ibm.com>

### CODE REVIEW

**Comment by cmluciano** [comment]
/assign @bowei @caseydavenport @dcbw 

Not sure if I missed any or added to jobs not owned by us

File: jobs/config.json [hunk 1]
```diff
@@ -3082,8 +3082,8 @@
       "--check-leaked-resources=true"
     ],
     "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "UNKNOWN"
+   "sigOwners": [
```
Comment (krzyzacy) [inline jobs/config.json]: the indention is wrong, and in other places. run jobs/config_sort.py will fix those.

Comment (cmluciano) [inline jobs/config.json]: thanks!

**Comment by caseydavenport** [comment]
This LGTM.

Thanks @cmluciano!

**Pull request merged**